bin/burden

```diff
@@ -2389,6 +2403,14 @@ convert_scenario_file()
 	if [ $? -eq 1 ]; then
 		cleanup_and_exit "The file, parse_file does not meet yaml requirements." 1
 	fi
+	#
```

Comment:
Fair warning: I'm going to start NAKing changes that include unrelated fixes because it makes finding and reverting breakages difficult.